webrtc-charter.html

```diff
@@ -233,6 +233,12 @@ <h3>
             <dd>JavaScript APIs that let a Web application manage how audio is rendered on the user audio output devices</dd>
             <dt><a href="http://w3c.github.io/mediacapture-screen-share/">Screen Capture</a></dt>
             <dd>An extension to the Media Capture and Streams API to use a user's display, or parts thereof, as the source of a MediaStream.</dd>
+	    <dt><a href="https://github.com/w3c/webrtc-quic">QUIC in WebRTC</a></dt>
+            <dd>An extension to the WebRTC 1.0 API to enable use of peer-to-peer QUIC streams for data exchange.</dd>
+            <dt><a href="https://github.com/w3c/webrtc-ice/">IceTransport extensions</a></dt>
+            <dd>An extension to the WebRTC 1.0 API to enable construction of RTCIceTransport objects.</dd>
+            <dt><a href="https://github.com/w3c/webrtc-dscp-exp/">Experimental QoS extensions</a></dt>
+            <dd>An experimental extension to the WebRTC 1.0 API to enable QoS marking.</dd>
```

Comment:
I don't think we should call it experimental in the charter - while it is experimental at the moment, presumably we would want to make it "real" during the charter.

Reply:
We'd want to run an experiment to figure out whether or not this is a control surface worth having?